test/unit/GridView.spec.js

```diff
@@ -0,0 +1,112 @@
+import { mount } from '@vue/test-utils'
+import Vue from 'vue'
+import GridView from '@/components/GridView.vue'
+import { CardTypes } from '@/types/Card'
+
+const generateSkeletons = (n) =>
+  Array(n).fill({
+    type: CardTypes.SKELETON
+  })
```

Comment:
@benjick Why does this function needs to be in the test specs?

Reply:
The function is in the `store/cards.ts` today. Maybe it'll be possible to move it to Grid.vue och GridView.vue at some point, hm.

The function is also a bit weird, the cards can be built using css instead (and being automatically duplicated)... https://css-tricks.com/building-skeleton-screens-css-custom-properties/